miniapps/convection/Particles3D/Layered_convection3D.jl

```diff
@@ -342,21 +340,21 @@ function main3D(igg; ar=1, nx=16, ny=16, nz=16, figdir="figs3D", do_vtk =false)
                 )
             end
 
-            xz_slice = ny >>> 1
+            slice_j = Int(ny ÷ 2)
```

Comment:
```suggestion
            slice_j = ny >>> 1
```

revert back to original notation